README.md

```diff
@@ -1,16 +1,131 @@
-# Title
+# Learning from Your Questions
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Identify effective methods of formulating questions
+- Identify community standards for writing help requests
 
 ## Introduction
 
-## SWBAT 1
+Before we get the right answer, we need to know how to ask the right question.
+Figuring out how to ask the right question is a process of clarifying exactly
+what we want the result to be as well as taking stock of all the information we
+currently have. The more thought we put into asking questions, the better chance
+we give ourselves of getting the right answer—either from others, or by
+ourselves.
 
-## SWBAT 2
+## Identify Effective Methods of Formulating Questions
+
+Sometimes, it's clear to us what we don't know and what we need to ask for. But
+often it's not so clear. Often, we don't know what we don't know and that makes
+it difficult to formulate a question to ask about it. In these not-so-clear
+cases, we can start with a process of determining what we _do_ know. Eventually,
+this process will lead us to the right question.
+
+Here's some steps to take when you're questions:
+
+1. __Focus on small section of code.__ Don’t keep scanning several files for a
+bug. Reducing the amount of material to sort through will allow you to isolate
+the issues faster and easier.
+
+2. __State your inputs.__ Double-check what data is getting pulled into your
```

Comment:
I think this is very good, but I think that we may need to consider moving this brick to a little later in the prework. Was talking to Steven about this - the beginning bricks of intro to programming assume the student is basically going in with nearly no knowledge of coding. While this is really good content for a new programmer, I think it still relies on the student having some understanding of input/output of a program or method, multifile programs, handling data, etc..

Reply:
nothing to fix here about it though. will bring it up in slack